packages/core/src/components/TextArea/TextArea.module.scss

```diff
@@ -0,0 +1,89 @@
+@import "../../styles/typography";
+@import "~monday-ui-style/dist/mixins";
+
+.textAreaWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: var(--spacing-xs);
+  @include smoothing-text;
+
+  .label {
+    @include vibe-text(text2, normal);
+
+    .required {
+      color: var(--color-error);
+    }
+  }
+
+  .helpText {
+    @include vibe-text(text2, normal);
+    color: var(--secondary-text-color);
+  }
+
+  .textArea {
+    resize: vertical;
+    border: 1px solid var(--ui-border-color);
+    border-radius: var(--border-radius-small);
+    padding: var(--spacing-small) var(--spacing-medium);
+    outline: none;
```

Comment:
It messes up the white space and I'm worried it might mess up more native properties, I rather not unset everything. Do you think there's something that worth unsetting seperately?

Reply:
hmm I'm not sure actually
you made me worried now about `BaseInput`'s `unset` 😬